### PR TITLE
Release 06 15 18 : Versioning changes

### PR DESCRIPTION
**Pull request by praries880**
Versioning Changes for the 6-15-18 release

### CODE REVIEW

**Comment by praries880** [comment]
on demand : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/934/
sign job : https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/743

File: src/ResourceManager/KeyVault/AzureRM.KeyVault.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '5.0.1'
+ModuleVersion = '6.0.0'
```
Comment (maddieclayton) [inline src/ResourceManager/KeyVault/AzureRM.KeyVault.psd1]: This shouldn't be a major version bump

File: src/ResourceManager/Profile/AzureRM.Profile.psd1 [hunk 2]
```diff
@@ -54,31 +54,31 @@ CLRVersion = '4.0'
 # RequiredModules = @()
 
 # Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = '.\Microsoft.Azure.Commands.Common.Authentication.Abstractions.dll', 
+RequiredAssemblies = 'Microsoft.Azure.Commands.Common.Authentication.Abstractions.dll',
```
Comment (maddieclayton) [inline src/ResourceManager/Profile/AzureRM.Profile.psd1]: Fix these

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -41,5 +41,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("4.1.3")]
-[assembly: AssemblyFileVersion("4.1.3")]
+[assembly: AssemblyVersion("4.2.0")]
```
Comment (maddieclayton) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Properties/AssemblyInfo.cs]: I think that the other recoveryservices.backup project need this change as well.

File: src/ResourceManager/Sql/AzureRM.Sql.psd1 [hunk 4]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '4.6.0'
+ModuleVersion = '4.7.0'
```
Comment (maddieclayton) [inline src/ResourceManager/Sql/AzureRM.Sql.psd1]: Not sure why this is a minor version bump based on the changelog

File: src/ResourceManager/Websites/AzureRM.Websites.psd1 [hunk 5]
```diff
@@ -141,8 +142,6 @@ PrivateData = @{
         # External dependent modules of this module
         # ExternalModuleDependencies = @()
 
-        # Prerelease = ''
```
Comment (maddieclayton) [inline src/ResourceManager/Websites/AzureRM.Websites.psd1]: Not sure why this is deleted, we probably want to keep this.

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Properties/AssemblyInfo.cs (hunk 3 above)
Comment (praries880) [reply src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Properties/AssemblyInfo.cs]: done

**Comment by praries880** [comment]
new sign job : https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/745/
on demand : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/935

**Comment by praries880** [comment]
one demand : on demand : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/936

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/749/

**Comment by praries880** [comment]
signed bits : \\aaptfile01\ADXSDK\PowerShell\2018-06-15-PowerShell

File: ChangeLog.md [hunk 6]
```diff
@@ -1,4 +1,63 @@
-﻿## 6.2.1 - June 2018
+﻿## 6.3.0 - june 2018
```
Comment (maddieclayton) [inline ChangeLog.md]: Can you fix the casing in the next RC?

File: tools/AzureRM/AzureRM.psd1 [hunk 7]
```diff
@@ -160,9 +160,64 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '6.2.1 - June 2018
-AzureRM.OperationalInsights
-* Updated PSWorkspace model to allow Network to use type as a parameter
+        ReleaseNotes = '6.3.0 - june 2018
```
Comment (maddieclayton) [inline tools/AzureRM/AzureRM.psd1]: Here too

File: ChangeLog.md (hunk 6 above)
Comment (praries880) [reply ChangeLog.md]: done

File: tools/AzureRM/AzureRM.psd1 (hunk 7 above)
Comment (praries880) [reply tools/AzureRM/AzureRM.psd1]: done

**Pull request merged**